src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,83 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+}
+
+.w-subscribe__border {
+  border: 1px solid $GREY_300;
+  width: 100%;
+
+  &:focus-within {
+    border-color: $WEB_PRIMARY_COLOR;
+    outline: none;
+  }
+}
+
+.w-subscribe__button {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 36px;
+}
+
+.w-subscribe__error {
+  color: $WARNING_COLOR;
+}
+
+.w-subscribe__field {
+  display: flex;
+  margin-top: 8px;
+}
+
+.w-subscribe__padded {
```

Comment:
Staring at this for a bit... I'm wondering if you need these classes? It seems like they're primarily used to center stuff. Could we use flex box to center align the three major sections and set a max-width on them as needed?